content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Quality Gates
+kind: documentation
+description: Learn how to use Quality Gates in your pipeline.
+is_beta: true
+further_reading:
+- link: "/continuous_integration/tests"
+  tag: "Documentation"
+  text: "Learn about Test Visibility"
+---
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+{{< callout url="#" header="false" >}}
+Quality Gates are in beta.
+{{< /callout >}}
+
+Quality Gates allows you to gate your workflows based on signals in Datadog.
+For example, you can block code from being merged if it introduces new [flaky tests][8], even if the tests are passing.
+
+Using Quality Gates, you have control over what is merged into the default branch, and ultimately on what is deployed.
+This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents and unwanted behaviors.
+
+## Set up Quality Gates
+
+There are two main steps required to set up Quality Gates:
+1. Create one or more rules.
+2. Add the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
+2. Select the rule type. You can choose between `Static Analysis` and `Test`.
+3. Define the rule scope. The rule scope defines when the rules should be evaluated. For example, you can scope a rule so that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope name. More information on [rule scopes](#rule-scope).
```

Comment:
```suggestion
3. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can scope a rule so that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope name. More information on [rule scopes](#rule-scope).
```